tests/test_add.py

```diff
@@ -214,3 +220,24 @@ def test_dir(self):
 
         ret = main(["add", self.DATA_DIR])
         self.assertEqual(ret, 0)
+
+
+class TestShouldUpdateStateEntryForFileAfterAdd(TestDvc):
+    def test(self):
+        file_md5_counter = spy(dvc.state.file_md5)
+        with patch.object(dvc.state, "file_md5", file_md5_counter):
+
+            ret = main(["config", "cache.type", "copy"])
+            self.assertEqual(ret, 0)
+
+            ret = main(["add", self.FOO])
+            self.assertEqual(ret, 0)
+            self.assertEqual(file_md5_counter.mock.call_count, 1)
+
+            ret = main(["status"])
+            self.assertEqual(ret, 0)
+            self.assertEqual(file_md5_counter.mock.call_count, 1)
```

Comment:
This should actually be 
```
self.assertEqual(file_md5_counter.mock.call_count, 0)
```
see https://github.com/iterative/dvc/pull/1603/files#r255385913 .